pkg/assets/assets/monitoring/system-grafana-dashboard-1.json.tpl

```diff
@@ -117,438 +128,80 @@
         "alignLevel": null
       }
     },
-    {
```

Comment:
So this should be kept. Do not remove the operations metrics

Reply:
This is also an old version of pod resources
Lets migrate it to the new version, copy the zync or backend dashboard and add the application metrics.

Reply:
If I understand this correctly, we want to the pods resource consumption(CPU-Usage/Quota, Memory-Usage/Quota, Network-Usage) distributed per Deployment Configs.